<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepMode</class>
 <widget class="QWidget" name="StepMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#F9F6F4 ;
color:#1B224D; 
font-family: 'Roboto', sans-serif;
font-size: 15px;
font-weight: bold;</string>
  </property>
  <widget class="QTableView" name="GeneratorTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QTableView" name="BufferTable">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QTableView" name="DeviceTable">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>231</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Generators</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Buffer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Devices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>470</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #F9F6F4;
 background-color:#1B224D; 
 font-size: 16px;
 border-radius: 0.5em;
 width: 12em;
 height: 3em;
 text-transform: uppercase;
 font-weight: bold;
  font-family: 'Roboto', sans-serif;
 text-align: center;</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>580</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #F9F6F4;
 background-color:#1B224D; 
 font-size: 16px;
 border-radius: 0.5em;
 width: 12em;
 height: 3em;
 text-transform: uppercase;
 font-weight: bold;
  font-family: 'Roboto', sans-serif;
 text-align: center;</string>
   </property>
   <property name="text">
    <string>Again</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #F9F6F4;
 background-color:#1B224D; 
 font-size: 16px;
 border-radius: 0.5em;
 width: 12em;
 height: 3em;
 text-transform: uppercase;
 font-weight: bold;
  font-family: 'Roboto', sans-serif;
 text-align: center;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DeniedRequest">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>490</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Denied Request</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Step</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StepField">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SetStepButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #F9F6F4;
 background-color:#1B224D; 
 font-size: 14px;
 border-radius: 0.5em;
 width: 12em;
 height: 3em;
 text-transform: uppercase;
 font-weight: bold;
  font-family: 'Roboto', sans-serif;
 text-align: center;</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
